api/src/uvisor-input.S

```diff
@@ -114,6 +114,9 @@ uvisor_config:
     /* Debug driver pointer */
     .long __uvisor_debug_driver
 
+    /* Stack limit for MSPLIM_NS */
+    .long __uvisor_stack_limit
```

Comment:
IMHO the variable name is misleading as it holds not the uvisor stack limit but the public box stack limit.
Am i missing something?

Reply:
fixed